src/js/containers/generateEF/GenerateEFContainer.jsx

```diff
@@ -154,10 +170,16 @@ class GenerateEFContainer extends React.Component {
 		if(this.props.submission.publishStatus !== "unpublished") {
 			warningMessage = <PublishedSubmissionWarningBanner />;
 		}
+
+		let gtasWarning = null;
+		if(this.state.gtas && this.state.gtas.open){
```

Comment:
gtas.open no longer exists. Given the suggestion below you can probably just do the first part of this if for checking.